tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_local_backend/artifacts/layer9/lib.py

```diff
@@ -0,0 +1,4 @@
+import boto3
```

Comment:
if we are not going to use boto3, is there a reason we import it?

Reply:
to make sure that dependencies are imported and the build process is done successfully